cmd/fyne/internal/util/file.go

```diff
@@ -55,7 +54,11 @@ func copyFileMode(src, tgt string, perm os.FileMode) error {
 	if err != nil {
 		return err
 	}
-	defer target.Close()
+	defer func() {
+		if r := target.Close(); r != nil {
```

Comment:
same here, could be simplified by doing just:
```go
err = target.Close()
return
```

Reply:
no, that would not work.

Reply:
You'd not return the error then, but just return from the deferred function, ignoring the error.

Reply:
You are right, I didn't realize it, sorry for the confusion. However, now I wonder if it could be rewritten to:
```go
if r := target.Close(); r != nil && err == nil {
     err = r
}
```
Otherwise, the error in the defer function will overwrite any previous error that maybe it is more helpful because it is the root cause of the problem?

Reply:
That is correct.

```suggestion
		if r := target.Close(); r != nil && err == nil {
```